htdocs/core/modules/stock/doc/pdf_standard.modules.php

```diff
@@ -362,15 +363,17 @@ public function write_file($object, $outputlangs, $srctemplatepath = '', $hidede
 						$showpricebeforepagebreak = 1;
 
 						$pdf->startTransaction();
-						pdf_writelinedesc($pdf, $object, $i, $outputlangs, $this->posxtva - $curX, 3, $curX, $curY, $hideref, $hidedesc);
+						$pdf->writeHTMLCell($this->wref, 3, $curX, $curY, $outputlangs->convToOutputCharset($objp->ref), 0, 1, false, true, 'J', true);
```

Comment:
Hello @eldy in this case, only the product reference should be printed, since there is another column for the label. So I simplified it by calling writeHTMLCell() directly. Also, pdf_writelinedesc() was doing nothing, as it was not being called with the correct parameters (in fact, it was reprinting the reference on line 420). This document is special since the variable $object is the warehouse (it does not have lines) and an SQL query is made to obtain the information of the products (variable $objp).